src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -2360,7 +2360,7 @@ protected override object GetChildItemsDynamicParameters(string path, bool recur
         /// <summary>
         /// get the name of the specified certificate
         /// </summary>
-        /// <param name="cert"> .</param>
+        /// <param name="cert"></param>
```

Comment:
We should probably change the other ones with a single space (or space with period) to just empty and (hopefully) get actual descriptions in over time